### PR TITLE
Dev ndraster + Atlas release + gene expression

### PR DESCRIPTION
**Pull request by jonathanlurie**

### CODE REVIEW

File: shapes/neurosciencegraph/datashapes/atlas/volumetricdatalayer/schema.json [hunk 1]
```diff
@@ -44,56 +46,30 @@
...
+              "description": "The modality of data capture (MRI, fMRI, EM, etc.)",
+              "datatype": "xsd:string",
+              "maxCount": 1
             },
             {
               "path": "nsg:brainLocation",
               "name": "brain location",
               "description": "Brain Location (brain region, atlas space).",
-              "node": "https://neuroshapes.org/commons/brainlocation/shapes/BrainLocationShape",
+              "node": "this:BrainLocationShape",
```
Comment (MFSY) [inline shapes/neurosciencegraph/datashapes/atlas/volumetricdatalayer/schema.json]: This should be https://neuroshapes.org/commons/brainlocation/shapes/BrainLocationShape

File: shapes/neurosciencegraph/datashapes/atlas/brainparcellationmesh/schema.json [hunk 2]
```diff
@@ -34,8 +34,8 @@
             },
             {
               "path": "nsg:brainLocation",
-              "name": "Soma brain location",
-              "description": "Brain Location (brain region, position In Layer, longitudinalAxis) of the reconstructed cell soma.",
+              "name": "Brain location",
+              "description": "Brain Location (brain region, position In Layer, longitudinalAxis) represented by the mesh",
```
[the remark this answers is not in the document]
Comment (jonathanlurie) [reply shapes/neurosciencegraph/datashapes/atlas/brainparcellationmesh/schema.json]: It's a schema for describing a mesh of a brain region. But yeah, I'll word that better

[the remark this answers is not in the document]
Comment (jonathanlurie) [reply shapes/neurosciencegraph/datashapes/atlas/brainparcellationmesh/schema.json]: It's a schema for a mesh of a brain region, so the "brain location" prop is directly related to what the mesh represents and where the vertices of this mesh are located

File: shapes/neurosciencegraph/datashapes/atlas/imagestack/schema.json [hunk 3]
```diff
@@ -38,24 +37,24 @@
...
             },
             {
-              "path": "nsg:sliceWidth",
-              "name": "Slice width",
-              "description": "Width of the 2D slice",
-              "node": "https://neuroshapes.org/commons/quantitativevalue/shapes/QuantitativeValueShape",
+              "path": "nsg:fixedSliceWidth",
+              "name": "Fixed Slice width",
+              "description": "Width of the 2D slices, to be provided only if all the slices are the same width within the stack.",
+              "datatype": "xsd:integer",
```
[the remark this answers is not in the document]
Comment (jonathanlurie) [reply shapes/neurosciencegraph/datashapes/atlas/imagestack/schema.json]: So I should have mentioned that this is in pixels, so we don't want to give the freedom of using another unit. I can update the description to reflect that.

File: shapes/neurosciencegraph/datashapes/atlas/imagestack/schema.json [hunk 4]
```diff
@@ -38,24 +37,24 @@
...
             },
             {
-              "path": "nsg:sliceHeight",
-              "name": "Slice height",
-              "description": "Height of the 2D slice",
-              "node": "https://neuroshapes.org/commons/quantitativevalue/shapes/QuantitativeValueShape",
+              "path": "nsg:fixedSliceHeight",
+              "name": "Fixed Slice height",
+              "description": "Height of the 2D slices, to be provided only if all the slices are the same width within the stack.",
+              "datatype": "xsd:integer",
```
[the remark this answers is not in the document]
Comment (jonathanlurie) [reply shapes/neurosciencegraph/datashapes/atlas/imagestack/schema.json]: So I should have mentioned that this is in pixels, so we don't want to give the freedom of using another unit. I can update the description to reflect that.

File: shapes/neurosciencegraph/datashapes/atlas/volumetricdatalayer/schema.json (hunk 1 above)
Comment (jonathanlurie) [reply shapes/neurosciencegraph/datashapes/atlas/volumetricdatalayer/schema.json]: Here `this:BrainLocationShape` only enforce a `"minCount": 1` to the original `https://neuroshapes.org/commons/brainlocation/shapes/BrainLocationShape`. I personally think it's good to enforce it, though in a near future, this dataset will be linked to an `AtlasRelease`.

Comment (MFSY) [reply shapes/neurosciencegraph/datashapes/atlas/volumetricdatalayer/schema.json]: Alright I see.

**Pull request merged**